### PR TITLE
toast component added

### PR DESCRIPTION
**Pull request by MonikaJhingan**
Please review urgently!!!

### CODE REVIEW

**Comment by Lokeshkhati** [comment]
code is neat and looks good  to me , can be merged

**Review by Lokeshkhati — APPROVED** [review]

File: Toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+.btn {
+  height: 2rem;
+  width: 10rem;
+  font-size: 2rem;
+  margin: 2rem;
+  padding: 2px 5px;
+  border:5px;
+  border-radius: 2 rem;
+  color: #e7eb0c;
```
Comment (RakshithBhat03) [inline Toast/toast.css:15]: You can use a variable here.

File: Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  background-color: black;
+  position: relative;
+  margin-left: 550px;
```
Comment (RakshithBhat03) [inline Toast/toast.css:18]: Use rem unit here, And why such a big margin?

File: Toast/toast.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  align-items:flex-end;
+}
+
+.toast {
+  font-size: 2.5rem;
+  border-radius: 8px;
+  padding: 1rem 1rem;
+  margin: 2rem;
+  width: 80rem;
+  margin-left: 200px;
```
Comment (RakshithBhat03) [inline Toast/toast.css:46]: Use rem here as well

File: Toast/toast.css [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+
+.toast-success {
+  background-color: rgb(15, 128, 0);
+  color: rgb(240, 225, 15);
```
Comment (RakshithBhat03) [inline Toast/toast.css:52]: You can use variables here as well

**Review by RakshithBhat03 — COMMENTED** [review]
Suggested a few changes. Please use variables wherever possible.

**Review by aamrapaliwandhre — APPROVED** [review]
Remove extra space from file ....Otherwise code looks good to me....good to merge

File: Toast/toast.css (hunk 1 above)
Comment (MonikaJhingan) [reply Toast/toast.css:15]: ok

File: Toast/toast.css (hunk 2 above)
Comment (MonikaJhingan) [reply Toast/toast.css:18]: ok

File: Toast/toast.css [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+.btn {
+  height: 2rem;
+  width: 10rem;
+  font-size: 2rem;
+  margin: 2rem;
+  padding: 2px 5px;
+  border:5px;
+  border-radius: 2 rem;
+  color: #e7eb0c;
+  background-color: black;
```
Comment (pariyar07) [inline Toast/toast.css:16]: Add color variables as much as possible

**Comment by pariyar07** [comment]
Add some comments on the code to make it easy for others to review. Anyways others look good. Please go ahead and merge.

**Pull request merged**